packages/standard-linter/index.js

```diff
@@ -6,29 +6,74 @@ module.exports = {
     "airbnb-typescript/base",
     "prettier",
   ],
-  plugins: ["simple-import-sort", "prettier"],
+  plugins: ["simple-import-sort", "check-file", "unused-imports", "prettier"],
   parser: "@typescript-eslint/parser",
   rules: {
     "@typescript-eslint/no-floating-promises": "error",
-    "@typescript-eslint/no-unused-vars": [
+    "no-console": "error",
+
+    /**
+     * Separates out the `no-unused-vars` rule depending on it being an import statement in the AST and providing
+     * an auto-fix rule to remove the nodes if they are imports.
+     * With this, we can now target test files with `'unused-imports/no-unused-vars': 'off'` for testing DX.
+     */
+    "@typescript-eslint/no-unused-vars": "off",
+    "unused-imports/no-unused-imports": "error",
+    "unused-imports/no-unused-vars": [
       "error",
       {
         argsIgnorePattern: "^_",
         varsIgnorePattern: "^_",
         caughtErrorsIgnorePattern: "^_",
       },
     ],
+
+    // import sort
     "sort-imports": "off",
     "import/order": "off",
     "simple-import-sort/imports": "error",
     "simple-import-sort/exports": "error",
+
+    // enforce PascalNaming for all files
+    "check-file/filename-naming-convention": [
+      "error",
+      {
+        "src/**/*.{js,jsx,tsx,ts}": "PASCAL_CASE",
+      },
+      {
+        ignoreMiddleExtensions: true,
+      },
+    ],
```

Comment:
this shouldn't apply to standard-linter, which doesn't use react right? we have files like `types`, `constants`, `helper` and this will force `BrowserWallets.types.tsx`, etc

Reply:
we can exclude standard namings for `types, constants, helper`  but i think its better to have a file name convention set out early on.

Reply:
for `BrowserWallets.types.tsx` case i think its alright for it to be pascalcase

Reply:
hmmm I'm skeptic about this rule

Reply:
we should not need to repeat the folder name, this will make import paths more verbose

Reply:
i mean if you dont want to repeat folder name can just leave as `types`, we can create an exception for that.

Reply:
as discussed with @thedoublejay , will remove this from standard and apply on project level instead. but for the dependency, still prefer it to be here 🤔

Reply:
i think the dependency should be removed because if it's project level, maintainers checking `standard` might think that the plugin being here will automatically be enabled (along with any default rules)

Reply:
well, this is why i wanted the rule to be here. if the rule is fragmented across projects then each project has to maintain the eslint deps as well 😢

Reply:
alright keep the dependency here, however add a note in README.md that this plugin is included but no rules are enabled by default

Reply:
i'll set it to off by default and each project can override themselves